solidity/contracts/ECDSARewards.sol

```diff
@@ -88,8 +88,22 @@ contract ECDSARewards is Rewards {
     uint256 internal constant minimumECDSAKeepsPerInterval = 1000;
 
     BondedECDSAKeepFactory factory;
-
-    constructor(address _token, address payable _factoryAddress)
+    TokenStaking tokenStaking;
+
+    // TODO: set actual value
+    uint256 internal constant operatorRewardCap = 400000 * 10**18;
```

Comment:
should the cap be for the beneficiary address rather than the operator address? 

0xABF7335F324873Ccb1960C9cDb7278E627dd39eF
received the highest rewards out of all the beneficiaries (i believe 882,885 KEEP) by running 4 nodes and only staking 416,668 KEEP across them by juicing large BTC lot sizes. They would be unaffected by the change

0xBF62A4cf0e78b8799dcA0B481E4669a760C0f99F
will receive 669,583 KEEP from running one node, staking 9,293,734 KEEP and bonding 3,750 ETH and not juicing the keeps. they have the highest number of keeps still open. They would be affected by the change

Reply:
Addressed in https://github.com/keep-network/keep-ecdsa/pull/584/commits/35c47b5dfd6fdcc0572050b117b554899f95de84.